### ISSUE TITLE
Dúvidas

### ISSUE THREAD

**Issue by bolokoz** [rromildosf/dental_radiograph_augmentation#1]
Olá,
estou dando uma pesquisada sobre data augmentation usando o imgaug também e achei seus scripts e gracas a eles pude entender melhor como a biblioteca funciona.
Eu estou querendo usar o object detection do tensorflow e ele possui uma ferramenta de data augmentation embutida mas não sei usar ela exatamente, aí tentarei produzir as augmentations por fora.

Posso saber qual framework (tensorflow, keras, torch) você está usando?
Que tipo de bounding box e como vc produziu elas também? Eu usei o LabelImg e ela gera arquivos xml, fica complicado manipular...

Obrigado
Att

**Comment by rromildosf** [comment]
Olá bolojoz
Eu fiz este pequeno "projeto" para um amigo meu que precisava usar em um projeto dele. Na verdade fiz duas versões, uma para tratar bounding boxes, que é esta, e outra para keypoints. Eu estou usando uma implementação do Mask Rcnn implementado no keras. Para marcar as imagens, no meu caso, usei o Via `http://www.robots.ox.ac.uk/~vgg/software/via/`, para realizar segmentação de 'objetos'. Ele é bem legal pois exportar em JSON e CSV, mas usei o json.

**Comment by bolokoz** [comment]
Valeu brother! Muito legal essa ferramenta 'Via'
No caso voce seguiu algum tutorial pra implementar o Mask Rcnn + Keras?

**Comment by rromildosf** [comment]
Na verdade estou utilizando a implementação feita pelo [Matterport](https://github.com/matterport/Mask_RCNN) para uns testes que estou fazendo.
